src/trait-bounds.md

```diff
@@ -73,6 +73,26 @@ fn name_figure<U: Shape>(
 }
 ```
 
+Bounds that don't use the item's parameters or [higher-ranked lifetimes] are
+checked when the item is defined. It is an error for such a bound to be false.
```

Comment:
Seems you're turning the second paragraph into "one sentence per line" but the first doesn't.

```suggestion
Bounds that don't use the item's parameters or [higher-ranked lifetimes] are checked when the item is defined.
It is an error for such a bound to be false.
```

Reply:
Thanks for the review! I pushed an update.